index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav>
+        <ul>
+            <li class="list-item">
+                <h1>My Logo</h1>
+            </li>
+            <li class="list-item"><img src="images/Union.png" alt="mobile navigation"></li>
+        </ul>
+    </nav>
+    <div class="headline">
+        <section class="main_content">
+            <h1 class="username">I’m Zewdie <br>Glad to see you!</h1>
+            <p class="introduction">I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </section>
+        <section class="social_links">
+            <h1 id="lets_connect">LET'S CONNECT</h1>
+            <article class="links">
+                <a href="#">
+                    <img src="/images/twitter.png" alt="">
+                </a>
+
+                <a href="#">
+                    <img src="/images/Linkedin.png" alt="">
+                </a>
+
+                <a href="#">
+                    <img src="/images/Vector@2x.png" alt="">
+                </a>
+
+                <a href="#">
+                    <img src="/images/crossfinger.png" alt="">
+                </a>
+            </article>
+        </section>
+    </div>
```

Comment:
- [ ] The container of the headline section is a `section` tag and it is the only section that has a section tag, so kindly remove the rest of the section tags and use a div tag instead of them.
- [ ] Kindly use the font family like the Figma design 
- [ ] Also the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link ( `a` tag )

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/212073438-0e5fa202-c13f-4133-9ff8-2be3c36062f3.png)   |  ![image](https://user-images.githubusercontent.com/83360636/212073517-e7651341-015d-4333-8b69-cdeea660d7e1.png)
  |